README.md

```diff
@@ -17,21 +9,36 @@ implemented doesn't correspond to any existing definition and is a
 simple `mnemonic op1, op2, ...` format. Functions are formatted in a
 way similar to how Google Chrome does in the debug console.
 
-### Installation
+## ❗ Important
 
-```
+This is a fork of the original project that the author was no longer able to
+spend time on: https://github.com/athre0z/wasm. The changes made here are
+primarily to support the
+[Manticore](https://github.com/trailofbits/manticore) project.
+
+New issues and pull requests will be reviewed on a best-effort basis. Please
+open an issue first if you think fixing the problem will be complex; this is so
+we can evaluate whether a fix or feature is in scope before comitting time to
+review. When opening an issue, please include information on how to reproduce
+what you are seeing. If you feel comfortable, please submit a well-crafted,
+minimal pull request that we can review.
+
+## Installation
+
+```sh
 # From PyPi
-pip install wasm
+pip install wasm-tob
 
 # From GitHub
-pip install git+https://github.com/athre0z/wasm.git
+pip install git+https://github.com/trail-of-forks/wasm-tob.git#egg=wasm-tob
```

Comment:
The `egg=` fragment is technically non-standard (and I've been working with the `pip` devs to remove it 😅) -- this should be equivalent:

```
pip install git+https://github.com/trail-of-forks/wasm-tob.git
```

(assuming that `wasm-tob` is at the top-level of the repo.)

Reply:
Good to know! I'll make the change